src/database-controller/sdk/.eslintignore

```diff
@@ -0,0 +1,7 @@
+# ESLint always ignores files in /node_modules/* and /bower_components/*.
+/build/**
+/deploy/**
```

Comment:
Why so many this file in your pr?

Reply:
It is for lint purpose. Because there are two `npm` package: `src` and `sdk`, so I have to add two of these files.